apps/mdn/mdn-aws/k8s/kumascript.deploy.yaml.j2

```diff
@@ -60,11 +41,16 @@ spec:
               value: "{{ NEW_RELIC_LOG }}"
             - name: NEW_RELIC_BROWSER_MONITOR_ENABLE
               value: "{{ NEW_RELIC_BROWSER_MONITOR_ENABLE }}"
-            - name: newrelic__license_key
+            - name: NEW_RELIC_LICENSE_KEY
               valueFrom:
                 secretKeyRef:
                   name: newrelic-secrets
                   key: NEW_RELIC_LICENSE_KEY
+            - name: REDIS_URL
+              valueFrom:
+                secretKeyRef:
+                  name: mdn-secrets
+                  key: redis-url-2
```

Comment:
I hope redis/2 is okay to use. In the local docker-compose file, 0, 1 and 3 were in use already. So I picked 2 for the kumascript cache.  Do we have any other kind of service using db 2 or is it actually free?

Reply:
This is OK to use. We're currently using 2 for Kuma caching, but all of the keys are prefixed accordingly, so it's safe to have both using that DB.